### PR TITLE
Add Xfer Records OP-1 Drum Utility 1.02

### PR DESCRIPTION
**Pull request by danielbayley**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/op-1-drum-utility.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'op-1-drum-utility' do
+  version '1.02'
+  sha256 'ad7acd315f65a3ee359dc5fab5e6554d588cc9e212d2416648471669b128dab4'
+
+  url 'https://xferrecords.com/product_downloads/34/freeware'
+  name 'OP-1 Drum Utility'
+  homepage 'https://xferrecords.com/freeware'
+
+  app 'OP1DrumUtilityOSX_102/OP-1 Drum Utility.app'
```
Comment (commitay) [inline Casks/op-1-drum-utility.rb]: Use `#{version.no_dots}` here

File: Casks/op-1-drum-utility.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  vst_plugin 'Op1DrumUtil.vst'
+
+  preflight do
+    system_command '/bin/cp', args: ['-r', '--', "#{staged_path}/OP1DrumUtilityOSX_102/OP-1 Drum Utility.app/Contents/Resources/Op1DrumUtil.vst", staged_path]
```
Comment (commitay) [inline Casks/op-1-drum-utility.rb]: Use fileutils.cp and `#{version.no_dots}` here

**Comment by commitay** [comment]
> Driver: Software to make a hardware peripheral recognisable and usable by the system. If the software is useless without the peripheral, it’s considered a driver.

Not sure if this should be submitted to `drivers`

File: Casks/op-1-drum-utility.rb (hunk 1 above)
Comment (danielbayley) [reply Casks/op-1-drum-utility.rb]: Ah good catch!

**Comment by commitay** [comment]
Closing in favour of https://github.com/caskroom/homebrew-drivers/pull/226

**Pull request closed without merging**